### PR TITLE
Using set(EXECUTABLE testweb3) to bring the CMake file for the libweb…

### PR DESCRIPTION
**Pull request by bobsummerwill**
…three tests in line with the other files.

This also fixes automated dependency graph generation.

### CODE REVIEW

**Comment by gavofyork** [comment]
build failing?

**Comment by bobsummerwill** [comment]
I can't see the logs, @gavofyork.   Please could you cut-and-paste one?   I assume they are all the same?
- @chriseth

File: libwhisper/WhisperPeer.cpp [hunk 1]
```diff
@@ -28,8 +28,7 @@ using namespace dev;
 using namespace dev::p2p;
 using namespace dev::shh;
 
-WhisperPeer::WhisperPeer(std::shared_ptr<Session> _s, HostCapabilityFace* _h, unsigned _i, CapDesc const&, uint16_t _capID):
```
Comment (subtly) [inline libwhisper/WhisperPeer.cpp:31]: Why the reversion?

Comment (bobsummerwill) [inline libwhisper/WhisperPeer.cpp:31]: Trying to work out where the break came.    It's not from my edits.

Build fails.  Undo.  Build fails.  Redo.   Build succeeds.
Sad face.   That took a lot of hours :-(
@LefterisJP @gavofyork @chriseth.

Anyway - end result is this PR is now ready for merge, fixing the solidity module name inconsistency and the missing set(EXECUTABLE testweb3), now with some bonus messy history.

**Comment by bobsummerwill** [comment]
Thanks, @LefterisJP, and also for the PR-for-my-PR!   Nearly there, eh?

**Pull request merged**